app/qml/ExternalResourceBundle.qml

```diff
@@ -97,6 +97,11 @@ Item {
          */
         property var confirmImage: function confirmImage(itemWidget, prefixToRelativePath, value) {
           if (value) {
+            console.log("TRY TO RESCALE" + value)
+            if (!__inputUtils.rescaleImage(value, __activeProject.qgsProject))
+            {
+              console.log("RESCALE FAILED" + value)
+            }
             var newCurrentValue = __inputUtils.getRelativePath(value, prefixToRelativePath)
             itemWidget.editorValueChanged(newCurrentValue, newCurrentValue === "" || newCurrentValue === null)
```

Comment:
No need for the `console.log`, it is not included in logs from clients and `rescaleImage` already logs when it fails
```suggestion
            if (__inputUtils.rescaleImage(value, __activeProject.qgsProject))
            {
              var newCurrentValue = __inputUtils.getRelativePath(value, prefixToRelativePath)
              itemWidget.editorValueChanged(newCurrentValue, newCurrentValue === "" || newCurrentValue === null)
            }
```